CMakeLists.txt

```diff
@@ -14,6 +14,14 @@ set(Clang_DIR_ORIG ${Clang_DIR})
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/Modules ${CMAKE_MODULE_PATH})
 SET(CMAKE_MODULE_PATH_ORIG "${CMAKE_MODULE_PATH}")
 set (CMAKE_MODULE_PATH "${LLVM_DIR};${Clang_DIR};${CMAKE_MODULE_PATH}")
+option(BINDER_ENABLE_TEST      "Enables building of tests" ON)
+option(BINDER_MOCK_TEST        "Fake running  binder on tests" OFF)
```

Comment:
Description should probably be ~"Mock running  binder..."

Reply:
Done.

Reply:
>  will tests run on all available pythons

Yes. All the pythons according to the specification.

3 -> the first found python of version 3.*
3.X -> the first found python of version 3.X.*
3.X.Y -> the first python of version 3.X.Y

Same for the version 2.

If there are two installations of the same python, one can prioritize one installation via the standard cmake variables.
( I don't remember them now, but these are described in cmake docs. )

>will remain installed on most systems probably forever. 

Yes. So as for me the defaults -DBINDER_TEST_PYTHON_VERSIONS=0,2,3   are reasonable.
Note that the order does not matter. 0,2,3 is the same as 3,2,0. Likely only the order of compilation will differ.

Reply:
👍 Great, - this sounds good to me!